.github/CODEOWNERS

```diff
@@ -94,3 +94,4 @@
 /src/walkabout                      @benesch
 /src/workspace-hack                 @benesch
 /test
+/misc/python/materialize/cloudtest  @MaterializeInc/cloud
```

Comment:
I would like to wait with this change until we actually move the code and then restrict the path to the specific subfolder that will contain the shared logic. In particular, I would not want this ownership for cloud _test cases_ that reside in the mz repo.

Furthermore, I would like @philip-stoev to sign this off.

Reply:
Is there a quick summary of what functions are going to be owned by the cloud team? Would it make sense to consider shared ownership between the cloud and QA teams for those functions?

Reply:
Yeah, I envision a shared ownership, so we should add the QA team as well if it is not listed yet.

The exact set of functions is still to be defined but will roughly be [tests/k8s/util (currently in cloud repo)](https://github.com/MaterializeInc/cloud/tree/main/tests/k8s/util).

Reply:
yes, I think the QA team should also be listed here (unless already inherited from some higher-level CODEOWNERS rule).